Increase # of works requested from Solr as collection size grows

nicktiberi opened the pull request:
Closes #1249.

To prevent users from seeing 'No results found' when searching for works to add to a collection, we now request 50 + the number of works that are currently in the collection. This ensures that if there are any eligible works to be added to the collection, we will show them in the dropdown without the user needing to type anything into the search box.

mtribone commented:
Using Prepend instead in another PR to alleviate unexpected behavior in the tests.

nicktiberi committed:
Increase # of works requested from Solr as collection size grows

To prevent users from seeing 'No results found' when searching for
works to add to a collection, we now request 50 + the number of works
that are currently in the collection. This ensures that if there are
any eligible works to be added to the collection, we will show them
in the dropdown without the user needing to type anything into the
search box.

banukutlu reviewed (approved):
lgtm 👍🏻
